components/FavCardTabView.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useCallback } from 'react';
+import { View, StyleSheet, Text } from 'react-native';
+import { TabView, SceneMap, TabBar } from 'react-native-tab-view';
+import COLORS from '../global-styles/Colors.js';
+import RecText from './RecText.js';
+import LineupText from './LineupText';
+
+const initialLayout = { width: 360, height: 360 };
+
+const useComponentSize = () => {
+  const [size,setSize] = useState({height: 360});
+
+  const onLayout = useCallback(event => {
+      const { width, height } = event.nativeEvent.layout;
+      setSize({ width, height });
+  }, []);
```

Comment:
seems like same function in another file. could be some merit to pulling this out either now or the next task that needs it